src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/services.component.ts

```diff
@@ -229,6 +243,15 @@ export class ServicesComponent extends ListWithDetails implements OnChanges, OnI
         context.error();
       }
     );
+    if (
+      this.isMgmtGateway &&
+      !this.services.find(
+        (service: CephServiceSpec) =>
+          service.service_type !== 'mgmt-gateway' && service.status.running > 0
+      )
+    ) {
+      this.isMgmtGateway = false;
+    }
```

Comment:
doubt: why is this?

Reply:
When mgmt-gateway service is enabled and running the services urls will change therefore, I'm checking here wether the service is active or not